setup.py

```diff
@@ -23,7 +23,7 @@ def _read_description():
 packages = [x for x in setuptools.find_packages() if x != 'test']
 setuptools.setup(
     name='outrank',
-    version='0.97.1',
+    version='0.97.2',
```

Comment:
Actually, a feature was just extended (a transformer namespace) + because backwards compat is guaranteed it makes sense to have it incrementally bumped